packages/hop-node/src/watchers/BondWithdrawalWatcher.ts

```diff
@@ -259,15 +259,11 @@ class BondWithdrawalWatcher extends BaseWatcher {
     logger.debug('transferNonce:', transferNonce)
     logger.debug('bonderFee:', this.bridge.formatUnits(bonderFee))
 
-    const dbTransfer = await this.db.transfers.getByTransferId(transferId)
-    const {
-      gasPrice,
-      tokenUsdPrice,
-      chainNativeTokenUsdPrice
-    } = await this.getPricesNearTransferEvent(dbTransfer)
-    logger.debug('gasPrice:', gasPrice.toString())
-    logger.debug('tokenUsdPrice:', tokenUsdPrice)
-    logger.debug('chainNativeTokenUsdPrice:', chainNativeTokenUsdPrice)
+    const minFee = BigNumber.from('1000000')
+    if (bonderFee.lt(minFee)) {
```

Comment:
I think this bonder fee check can be moved to the db getter so it doesn't try to process it at all